src/Management/src/MetricCollectors/Aggregations/IAggregationStatistics.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What is the benefit of separating into different files when we are trying to match the code from original source?

Reply:
I changed this before we decided to match the original source. I'm going to push another commit for that, which will revert this separation.

Reply:
I've pushed two commits: The first is the unmodified version from .NET 7 stable. The next one contains our modifications.